src/platforms/python/integrations/ariadne/index.mdx

```diff
@@ -27,9 +27,6 @@ from sentry_sdk.integrations.ariadne import AriadneIntegration
 
 sentry_sdk.init(
     dsn="___PUBLIC_DSN___",
-    # Set traces_sample_rate to 1.0 to capture 100%
-    # of transactions for performance monitoring.
-    traces_sample_rate=1.0,
     integrations=[
```

Comment:
Ariadne does not support performance monitoring as of now, so no need to activate it for this integration